src/dbus/abrt-polkit.c

```diff
@@ -90,22 +94,31 @@ static PolkitResult do_check(PolkitSubject *subject, const char *action_id)
     g_object_unref(auth_result);
     return result;
 }
+#endif
 
 PolkitResult polkit_check_authorization_dname(const char *dbus_name, const char *action_id)
 {
+#ifdef HAVE_POLKIT
     glib_init();
 
     PolkitSubject *subject = polkit_system_bus_name_new(dbus_name);
     return do_check(subject, action_id);
+#else
+    return PolkitYes;
+#endif
 }
 
 PolkitResult polkit_check_authorization_pid(pid_t pid, const char *action_id)
 {
+#ifdef HAVE_POLKIT
     glib_init();
 
     PolkitSubject *subject = polkit_unix_process_new_for_owner(pid,
             /*use start_time from /proc*/0,
             /*use uid from /proc*/ -1);
 
     return do_check(subject, action_id);
+#else
+    return PolkitYes;
```

Comment:
I would print out a warning message because this is harmful - everyone have access to private data of all other users.